### PR TITLE
[Enhancement] Macros for State.GetAll*

### PR DESCRIPTION
**Pull request by ruseinov**
## Summary of changes

Changes introduced in this pull request:

- Macros for `VerifiedRegistryStateExt`

## Reference issue to close (if applicable)

Work on #4550 

## Change checklist

- [x] I have performed a self-review of my own code,
- [ ] I have made corresponding changes to the documentation,
- [ ] I have added tests that prove my fix is effective or that my feature works (if possible),
- [ ] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by LesnyRumcajs — APPROVED** [review]

File: src/shim/actors/verifreg/state.rs [hunk 1]
```diff
@@ -249,109 +153,55 @@ impl VerifiedRegistryStateExt for State {
...
+            State::V11(state) => list_all_inner_old!(state, store, v11, load_claims, result),
+            State::V12(state) => list_all_inner!(state, store, v12, load_claims, result),
+            State::V13(state) => list_all_inner!(state, store, v13, load_claims, result),
+            State::V14(state) => list_all_inner!(state, store, v14, load_claims, result),
         };
         Ok(result)
     }
 }
+
+#[macro_export]
```
Comment (hanabi1224) [inline src/shim/actors/verifreg/state.rs:167]: Do we have to export these macros?

**Review by LesnyRumcajs — APPROVED** [review]

File: src/shim/actors/verifreg/state.rs (hunk 1 above)
Comment (ruseinov) [reply src/shim/actors/verifreg/state.rs:167]: If we don't - the macros have to be defined strictly above the code that uses them, which is inconvenient.

**Review by hanabi1224 — APPROVED** [review]

File: src/shim/actors/verifreg/state.rs (hunk 1 above)
Comment (aatifsyed) [reply src/shim/actors/verifreg/state.rs:167]: this exports the macro as a public item at the root of the crate. Does `pub(crate) use list_all_inner` below the macro work? Else I think moving the definition to the top of the file is the right thing to do

**Review by aatifsyed — DISMISSED** [review]

**Comment by ruseinov** [comment]
>this exports the macro as a public item at the root of the crate. Does pub(crate) use list_all_inner below the macro work? Else I think moving the definition to the top of the file is the right thing to do

Gotta `export` to `use`. Alright, let's move this to the top then :/

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**